src/Repository/UserRepository.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+
+namespace Marketredesign\MrdAuth0Laravel\Repository;
+
+use Auth0\SDK\API\Management;
+use GuzzleHttp\Exception\RequestException;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Cache;
+
+class UserRepository implements \Marketredesign\MrdAuth0Laravel\Contracts\UserRepository
+{
+    private const CACHE_TTL = 1800;
```

Comment:
I'd suggest putting this in the `config/mrd-auth0.php` config file with an env variable with a default value so that configuration constants are in one place and if needed can be altered with an env variable.

Reply:
Moved to config